index.html

```diff
@@ -59,6 +59,87 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        HTML allows the display of many media types through elements such as
+        &lt;img&gt;, &lt;picture&gt;, or &lt;video&gt;, but it does not provide
+        a native manner to directly consume 3D content. Embedding such content
+        within a page is comparatively cumbersome and relies on scripting the
+        &lt;canvas&gt; element. We believe it is time to put 3D models on equal
+        footing with other, already supported, media types.
+      </p>
+      <p>
+        There is a variety of solutions for displaying three-dimensional models
+        on the web, including general WebGL libraries and frameworks that have
+        the ability to parse and render models. There are also more
+        purpose-built libraries devoted to a simpler set of capabilities around
+        a single model, providing a simpler API to achieve this.
+      </p>
+      <p>
+        However, because these libraries constitute a layer of abstraction on
+        top of the browser, they are unreliable as a basis for browser- and
```

Comment:
I think this should be rephrased... at least, we can't say they are unreliable or anything. 

`<model>` is just an easy way to declaratively embed 3D stuff, with the benefit of some great privacy qualities that WebXR can't get access to the lighting... it doesn't compete with existing JS libraries. 

It also takes away all the pain of having to load a bunch of JavaScript. 

However, JS libraries to do 3D stuff (including things like model-viewer) are great and people who want to use that, totally should.

Reply:
I see what you're saying, I'll try to find a different wording. 

It's less that they're "unreliable" as libraries, but that as arbitrary JS abstractions,  they _cannot_ constitute a stable basis for direct API features. An OS or browser feature can't rely on a soft convention to do critical work like accessibility - it _must_ be formed around a standard that people agree on.

Reply:
yeah, makes sense... let's make sure that's more clearly reflected in the text.